### PR TITLE
Capstone Project: Conference page

### PR DESCRIPTION
**Pull request by sanieni6**
## First capstone Project: Conference Page (Quevedo Fest)
This is the first capstone project of the Microverve program corresponding to module one. 
This project was divided into three days of work:
### Day 1: 
+ Set up the repository and tools.  ✔️ 
+ Choose the topic and the resources for the project ✔️ 
+ Crete the mobile version ✔️ 
### Day 2:
+ Create the desktop version. ✔️ 
+ Implement the user interactions: link, mobile menu ✔️ 
### Day 3:
+ Implement the section "Featured speakers" with dynamic HTML. ✔️ 
+ Deploy the project ✔️ 
+ Record a video describing your project. ✔️

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        
+    </section>
+    <footer id="footer" class="footer">
+        
+        <img class="logo-footer" src="images/logofestivalbg.png" alt=" festival logo">
+        <div class="text-footer">
+        <h3 class="title-footer">2023 Alternative Events. <br> Some rigths reserved</h3>
+        <h4 class="description-footer">Quevedo fest is a registered branch of Alternative Events</h4>
+        </div>
+
```
Comment (m05e5) [inline index.html:148]: - [ ] Hi. As part of the requirement you are supposed to follow the provided design as much as possible so please try considering the below change suggestions
- The partner section should only be present on the home page not on the about page
- The footer on the about page is black. Please consider changing the color

**Provided design**
image


**your UI**
image

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @sanieni6 , 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights

- You truly did an amazing job with the User Interface (UI)

## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] As part of the best practices and to improve SEO ill recommend adding a `h1` tag you your app.
- [ ] Please check the comments below 

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sanieni6** [comment]
## Required changes done.
These were the changes done to the project:
+ Present the partner section just on the home page, not on the about page.*****
+ Change the color of the footer

****As part of the requirements in the Behance template, the partner section is not shown just in the about for the desktop version. It means the partner section will show in the about of the mobile version as you can see in the attached image.
image

**Review by TadesseDev — APPROVED** [review]
### Hi @sanieni6 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. ✅
 - [ ]  Correct GitHub Flow. ✅
 - [ ]  Good commit history. ✅
 - [ ]  Good PR title/description. ✅
 - [ ]  All linters passed ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**